addon/templates/components/polaris-breadcrumbs.hbs

```diff
@@ -15,7 +15,7 @@
   <button
     class="Polaris-Breadcrumbs__Breadcrumb"
     type="button"
-    onclick={{action breadcrumb.onAction}}
+    onclick={{action "fireAction" breadcrumb.onAction}}
```

Comment:
would it be the same result just doing `<button {{action breadcrumb.onAction}}...`?

Reply:
Yep! 👍 Going that way in the interest of cleaner code over matching the Polaris source exactly